index.bs

```diff
@@ -8892,17 +8892,12 @@ specified sequence of user input actions.
         ? pressure: float .default 0.0,
         ? tangentialPressure: float .default 0.0,
         ? twist: (0..359) .default 0,
-        (input.TiltProperties // input.AngleProperties)
-      )
-
-      input.AngleProperties = (
-         ? altitudeAngle: float .default 0.0,
-         ? azimuthAngle: float .default 0.0,
-      )
-
-      input.TiltProperties = (
-         ? tiltX: js-int .default 0,
-         ? tiltY: js-int .default 0,
+        ? tiltX: (-90..90) .default 0,
```

Comment:
But this seems wrong.

With this definition, if I write something like:
```js
{
  [….]
  tiltX: -45,
  tiltY: 45
}
```

Then I get something with `{tiltX: 45, tiltY: -45, altitudeAngle: 0, azimuthAngle: 0}` which is nonsense because it's not what you'd get from converting the specified `tiltX`, `tiltY` values into the corresponding `altitudeAngle` / `azimuthAngle` values. As a result we'll end up emitting an event with contradictory values, even though the user didn't do anything to cause that. This is strictly worse than the (already slightly broken) situation with classic where we only pass through the supplied values, and rely on the UA to default any missing values.

The original intent here was that you can either provide the `tiltX` and `tiltY` properties, _or_ `altitudeAngle` and `azimuthAngle`, but not both. It seems like we can't specify that easily in CDDL without resorting to named enums, which seems like overkill for this case. I can think of two obvious solutions:
* Don't specify defaults in CDDL, but have prose that specifies both the defaults and which combination of properties is allowed.
* Only support `altitudeAngle` and `azimuthAngle`, and require classic users to do the coordinate conversion if they're currently using `tiltX`/`tiltY`.

Reply:
SGTM

Reply:
We should probably also update the original WebDriver spec to specify how to handle this when generating the DOM events. But that doesn't block this.

Reply:
Resolved.